CRM/Utils/System/WordPress.php

```diff
@@ -863,4 +863,23 @@ public function synchronizeUsers() {
     ];
   }
 
+  /**
+   * Send an HTTP Response base on PSR HTTP RespnseInterface repsonse.
```

Comment:
typo

Reply:
Since you need to fix this comment block it's worth mentioning that the phpdoc std is a blank line after the first 'short sentence describing the function'